django_browserid/tests/test_views.py

```diff
@@ -116,6 +116,17 @@ def test_auth_fail(self):
         self.assertEqual(response.status_code, 403)
         self.assert_json_equals(response.content, {'redirect': '/fail'})
 
+    @mock_browserid(None)
+    def test_auth_fail_named_url(self):
+        """
+        If authentication fails, redirect to the failure URL, and resolve
+        named URLs.
+        """
+        with self.settings(LOGIN_REDIRECT_URL_FAILURE='epic_fail'):
```

Comment:
Could we add a new entry to the testing urlconf instead of reusing this? Specifically, I'd like the URL name and the actual path to differ to be more confident that it isn't just parsing it as a URL and adding in slashes.